src/pages/partners/partners.mdx

```diff
@@ -3,13 +3,13 @@ title: KubeEdge Partners
 hide_table_of_contents: true
 ---
 
-<span style={{fontSize:'18px', display:'block', marginBottom:'10px'}}>KubeEdge works with partners to create a strong, vibrant cloud native edge computing ecosystem. Here we have a list of <span style={{fontWeight:'bold'}}>KubeEdge Distributions, Hosted Platforms and Installers</span> with deep experience helping enterprises successfully adopt KubeEdge.</span>
+<span style={{fontSize:'18px', display:'block', marginBottom:'10px'}}>KubeEdge works with partners to create a strong, vibrant cloud native edge computing ecosystem. Here we have a list of <span style={{fontWeight:'bold'}}>KubeEdge Distributions, Hosted Platforms and Installers</span> with deep experience helping enterprises successfully adopt KubeEdge (Sort alphabetically by organization/company name).</span>
 
 
 <span style={{fontSize:'18px', display:'block', marginBottom:'20px'}}>To join this list, please follow <a href="https://kubeedge.io/docs/community/partners" target="_blank">partners instructions</a>.</span>
```

Comment:
```suggestion
<span style={{fontSize:'18px', display:'block', marginBottom:'20px'}}>To join this list (Sort alphabetically by organization/company name), please follow <a href="https://kubeedge.io/docs/community/partners" target="_blank">partners instructions</a>.</span>
```
How about add it after the list here?

Reply:
> How about add it after the list here?

Looks good. It's also added in partners instructions.